recipes/dragonbox/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class DragonboxConan(ConanFile):
+    name = "dragonbox"
+    description = "Reference implementation of Dragonbox in C++"
+    license = "Apache-2.0", "BSL-1.0"
```

Comment:
```suggestion
    license = ("Apache-2.0-with", "LLVM-exception", "BSL-1.0")
```
The LLVM exception is listed in Apache license file: https://github.com/jk-jeon/dragonbox/blob/master/LICENSE-Apache2-LLVM

According to the SPDX page, there is an official identifier for LLVM exception: https://spdx.org/licenses/LLVM-exception.html

Reply:
Thanks!